### ISSUE TITLE
save polls

### ISSUE THREAD

**Issue by patrickmohrmann** [mclemente/foundryvtt-easy-polls#11]
Hey, I would like to save my polls to reference it, once in a while

**Comment by mclemente** [comment]
You mean saving polls as a template to be used for later polls or a way to check old polls?

**Comment by patrickmohrmann** [comment]
yes exactly, I like to have polls after sessions for feedback, but do not want to write it everytime again

**Comment by mclemente** [comment]
I'll put this on hold until V11 release since it'll have the Persistent Storage feature, which will make it easier to share the polls between worlds and with other people.
Right now, it would require a setting and there wouldn't be a nice way to transfer them from world to world.

**Comment by Steelskinz** [comment]
> yes exactly, I like to have polls after sessions for feedback, but do not want to write it everytime again

(Little late but for other curious people)
Just do a Chat Macro with : 
/poll name
Option 1
Option 2
Option 3
Option 4
